customcomponent/src/components/Links.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
```

Comment:
Better to remove this import from react, Because you don't need it in the new version of react